src/main/java/com/example/schedule/demo/service/ScheduleImpl.java

```diff
@@ -1,13 +1,15 @@
 package com.example.schedule.demo.service;
 
 import com.example.schedule.demo.entity.Schedule;
+import com.example.schedule.demo.form.CreateForm;
 import com.example.schedule.demo.mapper.ScheduleMapper;
 import org.springframework.stereotype.Service;
+import org.springframework.validation.BindingResult;
 
 import java.util.List;
 
 @Service
-public class ScheduleImpl implements ScheduleService{
+public class ScheduleImpl implements ScheduleService {
```

Comment:
クラス名ですが、こちらの方が適切かなと...
```suggestion
public class ScheduleServiceImpl implements ScheduleService {
```

Reply:
修正しました。
https://github.com/aokiayukodesu/Schedule-list/pull/2/commits/eaafd075577d0377f5bbc51690ecb10a8afb965d